### ISSUE TITLE
Feature: Add property for remote with datatype boolean in JobPosting

### ISSUE THREAD

**Issue by sstankov** [schemaorg/schemaorg#1591]
Hi guys,

with more and more jobs are open to be worked from remote location, is good to add property that mark if this is possibility or not.

Keep in mind that sometimes remote is open for a country or region or other subset.

Cheers,
Slav

**Comment by vholland** [comment]
Rather than a boolean a new subtype like RemoteJobPosting may be better. That would allow us to add properties to specify things like the applicable country or region.

**Comment by EricAxel** [comment]
My preference is for property. There are jobs that are presented as in-office OR remote.

Additionally, could jobLocation not be used to describe a region that names the scope of "remote?"

Cheers,
--Eric

Eric Axel Franzon

**Comment by vholland** [comment]
A boolean would not let an author express in-office or remote.

Overloading jobLocation means you can't differentiate between "remote within the USA" and remote or USA.

**Comment by sstankov** [comment]
current there is:

`jobLocation:Place`

if we add new one:

`jobLocationRemote:Place`

if we have both, job will be allowed for on site and remote. If we have just one, we know what is the case.

I'm not sure if Place or PostalAddress is better. Pros for the first time is that we follow the same pattern, but PostalAddress is more accurate for remote usage.

**Comment by danbri** [comment]
Can anyone find examples of sites where the job's remoteness appears to be explicitly represented in their infrastructure?

**Comment by sstankov** [comment]
Hmm, not sure, but you can see sites like:

* https://remoteok.io or https://jobspresso.co/ or http://stackoverflow.com/jobs

But cases include:

No remote in a certain area:

<img width="1249" alt="2017-04-19_20-52-41" src="https://cloud.githubusercontent.com/assets/3425712/25196891/543565de-2542-11e7-8869-78faf28f9216.png">

Remote, without limiting the area:

<img width="795" alt="2017-04-19_20-55-05" src="https://cloud.githubusercontent.com/assets/3425712/25196982/9f5fac04-2542-11e7-819f-7f109c83e3c1.png">

**Comment by RichardWallis** [comment]
> if we add new one:
> 
> jobLocationRemote:Place

Remote could mean "anywhere on the planet" - how would that work as a _Place_ ?

**Comment by jvandriel** [comment]
> _"Remote could mean "anywhere on the planet" - how would that work as a Place ?"_

Wouldn't this work for that?
```
"jobLocationRemote":
{
  "@type":"Place",
  "name":"anywhere on the planet",
  "sameAs":"https://www.wikidata.org/wiki/Q2"
}
```

**Comment by danbri** [comment]
Anyone familiar with hropenstandards.org? I looked around a bit there and failed to find a treatment of remote work, but I'm sure the issue must have cropped up. I've asked for pointers [on Twitter](https://twitter.com/danbri/status/893129211447312384).

**Comment by vholland** [comment]
Resurrecting this thread. It would be nice to be able to describe the job location, but there is not a lot of consensus (even on the sites I looked at) about how to designate regions, percent of remote work allowed, etc.

Can we add a ```jobLocationType``` property with the expected type Text? If we start to see consensus on what people are using for text, we could create a more meaningful type with properties.

**Comment by vholland** [comment]
Created PR #2039 to add this to pending.

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
